src/app/grid/grid-filtering-template-sample/grid-filtering-template-sample.component.html

```diff
@@ -42,31 +42,11 @@
     </ng-template>
     <ng-template #dateFilterTemplate igxFilterCellTemplate let-column="column">
         <div class="filter-cell">
-            <igx-date-picker [value]="getFilterValue(column)"
-                             (valueChange)="onDateSelected($event, column)" mode="dialog">
+            <igx-date-picker [value]="getFilterValue(column)" (valueChange)="onDateSelected($event, column)" mode="dialog">
```

Comment:
In order to achieve the same appearance as the other three filter cells, we need to change the date picker template a bit more:
```
<igx-date-picker #dp [value]="getFilterValue(column)" (valueChange)="onDateSelected($event, column)" mode="dialog"
                [displayDensity]="'compact'" placeholder="Filter..." type="box">
                <igx-picker-toggle (click)="dp.open()" igxPrefix>
                    <igx-icon>search</igx-icon>
                </igx-picker-toggle>
                <igx-picker-clear *ngIf="value" igxSuffix (click)="dp.clear()">
                    <igx-icon>clear</igx-icon>
                </igx-picker-clear>
            </igx-date-picker>
```